test/e2e/fixture/certs/certs.go

```diff
@@ -0,0 +1,17 @@
+package certs
+
+import (
+	"path/filepath"
+
+	"github.com/argoproj/argo-cd/errors"
+	"github.com/argoproj/argo-cd/test/e2e/fixture"
+)
+
+func AddCustomCACert() {
```

Comment:
should this have `ClientCert` in the name?

Reply:
Nope. It adds a custom TLS CA certificate to the E2E server (I added E2E tests for `cert add-tls` to this PR as well)